pkg/api/install/install.go

```diff
@@ -89,7 +89,8 @@ func enableVersions(externalVersions []unversioned.GroupVersion) error {
 }
 
 // userResources is a group of resources mostly used by a kubectl user
-var userResources = []string{"rc", "svc", "pods", "pvc"}
+var userResources = []string{"cs", "configmaps", "ds", "deploy", "ep", "hpa", "ing", "jobs",
```

Comment:
no component status, no endpoints (not an 80% case), no limits, no nodes, keep pods spelled out for people.

@smarterclayton I'm on the fence for secrets and configmaps, you?